src/form/field/Time.mjs

```diff
@@ -202,7 +202,7 @@ class Time extends Picker {
      */
     beforeGetValueFormat(value, oldValue) {
         if (!value) {
-            this._valueFormat = value = new Intl.DateTimeFormat('de-DE', {
+            this._valueFormat = value = new Intl.DateTimeFormat(Neo.config.locale, {
```

Comment:
you miss the no 12h part :)

Reply:
done. Have to create branches.... that will make things cleaner if I have to do more than one edit